test/extended/single_node/topology.go

```diff
@@ -44,6 +51,30 @@ func getTopologies(f *e2e.Framework) (controlPlaneTopology, infraTopology v1.Top
 	return infra.Status.ControlPlaneTopology, infra.Status.InfrastructureTopology
 }
 
+// isInfrastructureStatefulSet decides if a StatefulSet is considered "infrastructure" or
+// "control plane" by comparing it against a known list
+func isInfrastructureStatefulSet(statefulSet appsv1.StatefulSet) bool {
+	infrastructureNamespaces := map[string][]string{
+		// No known OpenShift StatefulSets are considered "infrastructure" for now
+	}
+
+	namespaceInfraStatefulSets, ok := infrastructureNamespaces[statefulSet.Namespace]
+
+	if !ok {
+		return false
+	}
```

Comment:
(nit) shorter version
```suggestion
	namespaceInfraStatefulSets, _ := infrastructureNamespaces[statefulSet.Namespace]
```